README.md

```diff
@@ -0,0 +1,26 @@
+# Feedhenry Raincatcher GPS module
+
+This is a simple Raincatcher module that provides a service that subscribes and publishes location topics.
+
+## Topics
+
+### Subscription
+
+This module subscribes to the `wfm:gps:position:update` topic. This triggers a call to `navigator.geolocation` functionality in the browser if available.
```

Comment:
Nit: Should this be: "This triggers a call to **the** `navigator.geolocation` functionality in the browser if available?"